copis/gui/panels/controller.py

```diff
@@ -35,10 +36,12 @@ class ControllerPanel(scrolled.ScrolledPanel):
         parent: Pointer to a parent wx.Frame.
     """
 
-    def __init__(self, parent, *args, **kwargs) -> None:
+    def __init__(self, parent) -> None:
```

Comment:
Because `ControllerPanel` inherits from `scrolled.ScrolledPanel`, we should keep *args and **kwargs as we expect non-specified arguments (style, size, etc) to still work.

Reply:
Yeah but we aren't using those non-specified arguments anywhere. which means `*args` and `**kwargs` aren't needed.

Reply:
The [wxPython style guide](https://wiki.wxpython.org/wxPython%20Style%20Guide) recommends to "Use *args and **kwargs when subclassing wx.Windows", so we should keep it in.